### PR TITLE
Create Zero Networks SolutionMetadata.json

### PR DESCRIPTION
**Pull request by dicolanl**
Needed to creation solution package.

### CODE REVIEW

File: Solutions/ZeroNetworks/SolutionMetadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	"categories": {
+		"domains": [
+			"Security - Network"
+		],
+		"verticals": []
+	},
+	"support": {
+		"name": "Zero Networks",
+		"email": "support@zeronetworks.zendesk.com",
+		"tier": "community",
```
Comment (v-sabiraj) [inline Solutions/ZeroNetworks/SolutionMetadata.json:18]: @dicolanl, tier should be "Partner", thanks.

File: Solutions/ZeroNetworks/SolutionMetadata.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		"link": "https://github.com/Azure/Azure-Sentinel"
```
Comment (v-sabiraj) [inline Solutions/ZeroNetworks/SolutionMetadata.json:19]: @dicolanl, link should be the support link, thanks.

Comment (dicolanl) [reply Solutions/ZeroNetworks/SolutionMetadata.json]: fixed

**Comment by dicolanl** [comment]
@v-sabiraj fixed per comments

**Review by sreedharande — APPROVED** [review]

**Pull request merged**